test/test_suites.py

```diff
@@ -80,8 +93,9 @@ def create_end_to_end_test_suites(driver, nameSalt, schemaRegistryAddress, testS
     :return:
     '''
     test_suites = OrderedDict([
+        # Disable Failing tests only in confluent because of fips error, re-enable: SNOW-774533
```

Comment:
nit: rebase master, this will disable the test, I enabled it in more recent commits.

Reply:
reverted, unsure how that snuck in 😅